lib/mix/lib/mix/tasks/test.ex

```diff
@@ -175,6 +175,24 @@ defmodule Mix.Tasks.Test do
   def run(args) do
     {opts, files} = OptionParser.parse!(args, strict: @switches)
 
+    if opts[:listen_on_stdin] do
+      try do
+        Process.flag(:trap_exit, true)
```

Comment:
Maybe I'm missing something, but wouldn't all exceptions be logged by the code below?